src/infra/main/template.yaml

```diff
@@ -40,6 +40,20 @@ Parameters:
     Description: The name of the Core VC Storage stack
     Type: String
     Default: ais-core
+  AlertingSNSHighAlertNotificationTopicARN:
```

Comment:
think this should be low alert - reason being that we should not alert Live Services if a rollback is being triggered nor should we alert on single occurrence of error - the alarms in the Alarm template does this when it severe enough

Reply:
Okay yeah that makes sense to me,  will change them all over to Low Alerts instead and then I guess live services wouldn't need to take any action, but more so would be us 👍